netlogo-gui/src/main/app/tools/LibrariesTab.scala

```diff
@@ -3,11 +3,11 @@
 package org.nlogo.app.tools
 
 import java.awt.{ BorderLayout, Component, Dimension, FlowLayout, GridBagConstraints, GridBagLayout, GridLayout,
-                  Insets }
+  Insets }
 import java.awt.font.TextAttribute
 import java.io.IOException
 import java.nio.file.Path
-import javax.swing.{ Action, Box, DefaultListModel, JLabel, JList, JPanel, ListCellRenderer, ListModel }
+import javax.swing.{ Action, Box, DefaultListModel, JLabel, JList, JOptionPane, JPanel, JScrollPane, JTextArea, JTextField, ListCellRenderer, ListModel }
```

Comment:
Looks like these got accidentally added because I changed some GUI stuff after you created your branch. Make sure that the changes shown in the PR match what you intended locally.

Reply:
After double checking my branch, the import has been like that since the beginning. I also get "not found" errors when I removed the imports for JOptionPane, JScrollPane, JTextArea, and JTextField.

Reply:
That's strange, is your copy of the laf-widget branch up to date? My copy of `LibrariesTab` doesn't have those imports, and it compiles.

Reply:
You're correct, those imports weren't necessary. I forgot to pull earlier.